ftw/publisher/sender/utils.py

```diff
@@ -136,8 +136,13 @@ def get_site_relative_path(obj):
     obj_path = '/'.join(obj.getPhysicalPath())
     portal_path = '/'.join(portal.getPhysicalPath())
 
+    path = obj_path[len(portal_path):]
+    if path == '':
```

Comment:
You could also change that to `if obj_path == portal_path`, which is more expressive.